cmd/gaia/cli_test/test_helpers.go

```diff
@@ -14,10 +14,11 @@ import (
 
 	cmn "github.com/tendermint/tendermint/libs/common"
 
-	"github.com/cosmos/cosmos-sdk/client/keys"
+	clientkeys "github.com/cosmos/cosmos-sdk/client/keys"
```

Comment:
I remember to have seen this import being aliased with `clkeys` somewhere else in the codebase. It'd be nice if we could stay consistent. Not a blocker though.